RepoDb/RepoDb/QueryBuilder.cs

```diff
@@ -78,7 +88,7 @@ public QueryBuilder<TEntity> WriteText(string text)
 
         private QueryBuilder<TEntity> Append(string value)
         {
-            m_stringBuilder.Append(m_stringBuilder.Length > 0 ? string.Concat(" ", value) : value);
+            m_stringBuilder.Append(string.Concat(" ", value));
```

Comment:
@calumchisholm - this is an awesome findings by removing the checks in every append. I saw how you drop this leading space in the 'GetString()' method as well.